docs/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+    |--------------|------------------|----------------------------|
+    | States       |                5 | 2020-04-02 14:10:59.119746 |
+    | Contacts     |                0 | -                          |
+    | Comments     |                0 | -                          |
+    | Platforms    |                1 | 2020-04-02 14:10:54.507565 |
+
+*Note:* The dates will be different in your example, but the number of rows should be the same.
+
+5. Check the database status using Pepys Admin
+----------------------------------------------
+
+Run *Pepys Admin* from the Start Menu (you can either navigate to the *Pepys* folder and choose
+*Pepys Admin* or just search for it and press enter).
+
+The Pepys Admin command-line window will appear. Note that it shows the welcome banner, and states
+which database it is connecting to (this should be the same database that you configured in the
+configuration file). Choose option :code:`(2) Status`. Summary tables will be displayed showing the
+number of rows of different types in the database: it should show that there are 5 States, plus a
+single Sensor, Platform, Datafile and Change, and 15 Changes.
+
+6. Export a data file using Pepys Admin
+---------------------------------------
+
+Now choose option :code:`(4) Export by Platform and sensor`. You will need to select a platform -
+there is only one, so just press Enter. It will then give you a list of sensors and the periods they
+were active. Again there is only one, so just enter :code:`1` and press Enter. Press Enter to accept
+the default value for the output filename. You should then see a message saying :code:`Objects
+successfully exported to exported_GPX.rep.`. Exit Pepys Admin by choosing option :code:`(0) Exit`.
+
+If you open the file :code:`exported_GPX.rep` in a text editor, you should see contents like the
+following:
+
+.. code-block:: none
+
+    120427 162938.000	NELSON	AA	22 11 10.63 N	021 41 52.3702 W	0	8.75	0.0
+    120427 163038.000	NELSON	AA	22 17 10.63 N	021 47 52.3702 W	0	10.69	0.0
+    120427 163138.000	NELSON	AA	22 29 10.63 N	021 17 52.3702 W	0	14.58	0.0
+    120427 163238.000	NELSON	AA	22 11 10.63 N	021 35 52.3702 W	0	18.47	0.0
+    120427 163338.000	NELSON	AA	22 23 10.63 N	021 11 52.3702 W	0	6.80	0.0
+
+7. Import a file with errors
+----------------------------
+
+First, copy the entire :code:`track_files` folder from :code:`tests\sample_data` in the Pepys
+installation folder to a safe place - as the example we're going to run will move files.
+
+Open the Windows Command Prompt (open the *Start Menu* and type :code:`cmd` and press Enter) and
+use the :code:`cd` command to navigate to the Pepys install folder. Then navigate to the :code:`bin`
+directory and run :code:`set_paths.bat`. Move back to the main Pepys install folder with :code:`cd ..`,
+and run the following:
+
+.. code-block:: none
+
+    python -m pepys_import.import --path .\tests\sample_data\track_files\rep_data\rep_test1_bad.rep --resolver default --archive
```

Comment:
@robintw - this should be the path for the new folder. If we suggest a folder-name to use, we can use that folder in this example.

Reply:
Fixed in d8fbe92